ksqldb-parser/src/main/java/io/confluent/ksql/parser/properties/with/CreateSourceAsProperties.java

```diff
@@ -154,6 +155,22 @@ public Map<String, String> getKeyFormatProperties(
       builder.put(ProtobufProperties.UNWRAP_PRIMITIVES, ProtobufProperties.UNWRAP);
     }
 
+    final String nullableRep = props.getString(
+        CommonCreateConfigs.KEY_PROTOBUF_NULLABLE_REPRESENTATION);
+    if (ProtobufFormat.NAME.equalsIgnoreCase(keyFormat) && nullableRep != null) {
+      switch (ProtobufNullableConfigValues.valueOf(nullableRep)) {
+        case WRAPPER:
+          builder.put(ProtobufProperties.NULLABLE_REPRESENTATION,
+              ProtobufProperties.NULLABLE_AS_WRAPPER);
+          break;
+        case OPTIONAL:
+          builder.put(ProtobufProperties.NULLABLE_REPRESENTATION,
+              ProtobufProperties.NULLABLE_AS_OPTIONAL);
+          break;
+        default:
```

Comment:
Throw an exception here, to indicate that this should never happen?

Reply:
Done